x-pack/plugins/apm/public/components/app/tutorials/commands/node.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import {
+  serviceNameHint,
+  secretTokenHint,
+  serverUrlHint,
+  serviceEnvironmentHint,
+  apiKeyHint,
+} from './shared_hints';
+
+export const nodeVariables = {
+  apmServiceName: 'serviceName',
+  secretToken: 'secretToken',
+  apiKey: 'apiKey',
+  apmServerUrl: 'serverUrl',
+  apmEnvironment: 'environment',
+};
+
+export const nodeHighlightLang = 'js';
+
+const nodeServiceNameHint = i18n.translate(
+  'xpack.apm.tutorial.nodeClient.createConfig.commands.serviceName',
+  {
+    defaultMessage: 'Overrides the service name in package.json.',
+  }
+);
+
+export const nodeLineNumbers = (apiKey?: string) => ({
+  start: 1,
+  highlight: '3, 5, 7, 9',
+  annotations: {
+    3: `${serviceNameHint} ${nodeServiceNameHint}`,
+    5: apiKey ? apiKeyHint : secretTokenHint,
+    7: serverUrlHint,
+    9: serviceEnvironmentHint,
+  },
+});
+
+export const node = `// ${i18n.translate(
+  'xpack.apm.tutorial.nodeClient.configure.commands.addThisToTheFileTopComment',
+  {
+    defaultMessage:
+      'Add this to the very top of the first file loaded in your app',
+  }
+)}
+var apm = require('elastic-apm-node').start({
+  ${nodeVariables.apmServiceName}: '{{{apmServiceName}}}',
+
+  {{#apiKey}}
+  ${nodeVariables.apiKey}: '{{{apiKey}}}',
+  {{/apiKey}}
```

Comment:
My templating foo is rusty so I'm probably wrong but I was hoping we could simply do:

```suggestion
  apiKey: '{{{apiKey}}}',
```

(I also replaced the variable for apiKey)

Reply:
Reading the docs, it looks like `{{#apiKey}}...{{/apiKey}}` is for making it conditional. We don't want it to be conditional, do we?

Reply:
Ah, that makes sense 👍